docs/examples/debugging-workflows.md

```diff
@@ -0,0 +1,41 @@
+# Debugging workflows
+
+Sometimes you may need to connect to a running Fireactions VM in order to debug some workflow steps or look around in the host filesystem.
+
+[tmate](https://github.com/mxschmitt/action-tmate) action provides a way to SSH into a running runner VM and have full access.
+
+Using it is as easy - just decide which workflow step to intercept, comment out the following steps and include tmate action above it:
+
+```
+    - name: Setup tmate session
+      uses: mxschmitt/action-tmate@v3
+```
+
+For security purposes, it's advised to add SSH keys to your github [profile](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account) and limit access to yourself:
```

Comment:
```suggestion
For security purposes, it's advised to add SSH keys to your GitHub [profile](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account) and limit access to yourself:
```

Also, this could be a note, use this formatting:
```
!!! warning
```